src/app/app.component.ts

```diff
@@ -169,4 +171,18 @@ export class AppComponent implements OnInit, OnDestroy {
       }
     });
   }
+
+  /** Verifies that localStorage can be written to and read from */
+  private testLocalStorage() {
+    const testValue = 'testValue';
+    const testKey = 'testKey';
+    let retrievedValue: string;
+
+    localStorage.setItem(testKey, testValue);
```

Comment:
I think adding wrapping this in a try-catch would be safer?
Also, should we also test if `localStorage` exists?

Reply:
I will add the test of existence of `localStorage`.  As to wrapping this in `try-catch`, that seems to defeat the purpose of a global error handler.  What would we do in the `catch` besides re-throwing the exception?

Reply:
Maybe this could be in the `core/local-storage.service` ?

Reply:
@tomastrajan will do!